polkadot/xcm/procedural/tests/ui.rs

```diff
@@ -28,5 +28,5 @@ fn ui() {
 	std::env::set_var("SKIP_WASM_BUILD", "1");
 
 	let t = trybuild::TestCases::new();
-	t.compile_fail("tests/ui/*.rs");
+	t.compile_fail("tests/ui/*/*.rs");
```

Comment:
Does this make it only test the files 1 directory deeper, or does it also test the files under the `tests/ui` directory?

Reply:
It only worked for files 1 directory deeper, fixed it by using **